IFC4x3/PropertyConstants/s/SBB_MAINTENANCE_3TGYuHNSr6fB9CCE$kTIxO.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="SBB_MAINTENANCE" UniqueId="dd422e11-5dcd-46a4-b24c-30efee752ed8" />
+
```

Comment:
Delete (replaced by MAINTENANCE?)

Reply:
This is still present. Just removing it from the enum does not delete it.

Reply:
Resolved: https://github.com/bSI-InfraRoom/IFC-Specification/pull/180/commits/99abf8676afe057628b3d15628f7615fe1f2d645